src/main/fc/fc_tasks.c

```diff
@@ -208,6 +209,16 @@ void taskVtxControl(uint32_t currentTime)
 }
 #endif
 
+#ifdef CAMERA_CONTROL
+void taskCameraControl(uint32_t currentTime)
+{
+    if (ARMING_FLAG(ARMED))
```

Comment:
There is the potential for a race condition in here, if the craft is armed while a camera keypress is still active, it won't be terminated until the craft is disarmed.